src/main/resources/tsv-schema.json

```diff
@@ -21,7 +21,7 @@
   { "name": "collection method", "valueType": "string" },
   { "name": "host (scientific name)", "valueType": "string" },
   { "name": "host disease", "valueType": "string" },
-  { "name": "host age", "valueType": "string" },
+  { "name": "host age", "valueType": "number" },
```

Comment:
This was found after the bug was fixed. This is supposed to be number, but the bug never surfaced it.